P7 dwf2024 2

alexysge opened the pull request.

alexysge committed:
Agregando P4

alexysge committed:
Delete app.component.html

alexysge committed:
Delete styles.css

alexysge committed:
Update styles.css

alexysge committed:
Update app.component.html

alexysge committed:
Merge pull request #1 from alexysge/p5-dwf2024-1

P5 dwf2024 1

alexysge committed:
add src

cambiando index, main y styles

alexysge committed:
src/app P6

Agregando y actualizando para la p6

alexysge committed:
add angular.json

alexysge committed:
catergory component css y html

alexysge committed:
practica 7 terminada

alexysge committed:
P7 finalizada

Me falto un archivo

monmm reviewed (changes requested):
Hola!
Tienes 8 por la entrega tardía.

Por favor, resuelve el [pull request de la práctica 6](https://github.com/alexysge/alexysge-Proyecto-Desarrollo-Front_2024-2/pull/2) antes de aceptar este :)

alexysge committed:
Ng for en html

alexysge committed:
Correción de archivo

Tenia que subir el archivo en la rama de la p6 y la subi en la p7 pero ya con este commit vuelve a estar correcto
